v2/concordium/types.proto

```diff
@@ -2132,3 +2132,96 @@ message Branch {
   // Further blocks branching of this block.
   repeated Branch children = 2;
 }
+
+// ---------- Node Status ----------
+
+// The local time of the node given by
+// a unix time stamp (seconds).
+message UnixTimestamp {
+  uint64 timestamp = 1;
+}
```

Comment:
We already have a type `Timestamp` in millis, could we use this here as well?

Reply:
I suppose there's no harm in some extra precision :)